tests/e2e/integration/auth.spec.js

```diff
@@ -0,0 +1,37 @@
+import { URL } from '../config'
+
+const testKeyPath = 'test@notary.priv'
+
+describe('Test login page', () => {
+  it('Log in', () => {
+    cy.visit(URL)
```

Comment:
I noticed it reviewing #35; we can use `cy.visit('/')` instead because, according to [Cypress environment variables docs](https://docs.cypress.io/guides/guides/environment-variables.html#), cy.visit is prefixed with `baseUrl` 😃